### DISCUSSION TITLE
How-to update Zoraxy LXC? (There was a big update just recently)

### DISCUSSION THREAD

**Discussion by AdmiralAwesome** [tteck/Proxmox#1744] [Help Needed]
Hey,

Zoraxy is really cool and it had just recently a big update https://github.com/tobychui/zoraxy.
My question is - how can I update it manually? Or better yet - will it be possible to use awesome proxmox scripts to do it automagically? :)

**Comment by tteck** [comment]
That's a good question. Although I don't personally utilize Zoraxy, I need to install it in order to determine an update path. I suggest initiating by creating a backup of the LXC and then attempting the following steps:
```
cd /opt/zoraxy
systemctl stop zoraxy
git pull
cd /opt/zoraxy/src
go mod tidy
go build
systemctl start zoraxy
```

**Reply by AdmiralAwesome** [reply]
Awesome. I'll try to do it later today and post the result.

**Comment by AdmiralAwesome** [comment]
Worked like a charm.

```
cd /opt/zoraxy
systemctl stop zoraxy
git stash
git pull
cd /opt/zoraxy/src
go mod tidy
go build
systemctl start zoraxy
```

Before `git pull` execute `git stash`.

**Reply by AdmiralAwesome** [reply]
Just letting you know that update to latest (2.6.8) worked as well. Done just a few minutes ago. :)
